### PR TITLE
feat: generate todo Modal

### PR DESCRIPTION
**Pull request by GuptaEesh**
- add todo name
- add todo description
- set todo duration
- buttons to add the info and close the modal

todo-modal

### CODE REVIEW

File: src/Screens/focus-screen.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
...sssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssssdsssssssssssssssssssssssssssssss{" "}
```
Comment (AnuragSahu11) [inline src/Screens/focus-screen.js:35]: Is this a typo? if it is then correct it.

**Review by AnuragSahu11 — APPROVED** [review]
Code looks good , good variable names , nice folder structure. You can merge

**Review by abhij1607 — APPROVED** [review]
Neat n Clean Code
Liked the atomic components
good implementation in Modular code

File: src/Screens/focus-screen.js (hunk 1 above)
Comment (GuptaEesh) [reply src/Screens/focus-screen.js:35]: It was a testing typo ;( . I was checking the text overflow. Will surely remove it once I am done with all the features.

**Pull request merged**